### ISSUE TITLE
[Feature Request] Weapon jamming

### ISSUE THREAD

**Issue by Weebyxanide** [Regalis11/Barotrauma#2580]
When the only personal ranged weapons were spearguns and revolvers (alien weapons don't count - they're tools), spread alone was an adequate method of giving weapon skill a relation to the actual handling of them - if rarely relevant due to engagement ranges typically being contact shots. With the addition of several new weapons, though, some of them HIGHLY dangerous and indiscriminate, spread alone doesn't work. If anything, it lets grumpy people spread out the damage.

To cut down on just handing out such destructive weaponry, I propose adding a chance for a weapon to jam if one attempts to fire it without adequate skill, perhaps increasing the further they are from the minimum skill - with simpler things like the speargun and revolver being more idiot-proof. If a weapon jams, it must be fixed (with current mechanics this could just be a crafting operation - zero its condition when it rolls a malfunction, becomes broken weapon item what needs restored with weapons skill) to be able to fire again. This could probably be similar to onFail.

IN THEORY this should reduce the instances of assistants with no skill doing security's job better than them, without necessarily preventing heroics by completely taking firearms off the table. It gives weapons skill greater importance in personal combat, and security would probably be happy to strengthen their monopoly on violence ~~not that anyone likes those people~~

just a thought, imho, etc.

**Comment by Regalis11** [comment]
I agree, this would be a really good way to make the weapon skill have a bigger impact on how well you can use weapons. Closing this ticket and copying it to our private repo as we generally avoid having feature requests in the public issue tracker.
